spec/controllers/miq_report_controller/reports/editor_spec.rb

```diff
@@ -159,6 +159,39 @@
         end
       end
     end
+
+    describe "#categories_hash" do
+      it "returns non-empty categories" do
+        FactoryGirl.create(:classification_cost_center_with_tags)
+        FactoryGirl.create(:classification_department)
+        expect(controller.send(:categories_hash)).to eq("cc" => "Cost Center")
+      end
+
+      it "omits show: false categories" do
+        FactoryGirl.create(:classification_cost_center_with_tags, :show => false)
```

Comment:
Honestly, a toplevel `let(:classification) { FactoryGirl.create(...) }` that you then do `classification.update(:show => false) here would be helpful.